app/index.template.html

```diff
@@ -34,6 +34,12 @@
     </script>
   </head>
   {{#unless (ispage name 'tree')}}
-  <body data-notebook="{{ name }}"></body>
+  <body data-notebook="{{ name }}">
+    <noscript>
+      <div style="text-align: center; padding: 20px;">
+          JupyterLite requires JavaScript to be enabled in your browser.
+      </div>
+  </noscript>
+  </body>
```

Comment:
Dumb question, but should this be outside of the `{{#unless (ispage name 'tree')}}` ? I'm guessing tree also does not work ?

Reply:
Yeah maybe, I don't recall why we have this check (but could likely be found in the github history).